lib/setup.rb

```diff
@@ -6,120 +6,69 @@
 
 
 class Setup
-  attr_reader :computer,
+  attr_reader :ships,
               :player,
-              :computer_board,
-              :player_board,
-              :ships
-
+              :computer
 
   def initialize
     @computer_board = Board.new
     @player_board = Board.new
     @computer = Computer.new(@computer_board)
-    @player = Player.new(@player_board)
-
-  end
-
-  def collect_ships
-    @ships = []
-    @ships << Ship.new("Cruiser", 3)
-    @ships << Ship.new( "Submarine", 2)
+    @player = Player.new("Jason")
   end
 
   def give_ships
-    collect_ships
-    @player.ships = @ships
-    collect_ships
-    @computer.ships = @ships
+    players = [@computer, @player]
+    players.each do |player|
+      @ships = [Ship.new("Cruiser", 3), Ship.new( "Submarine", 2)]
+      player.ships = @ships
+    end
   end
 
-
-
   def computer_place_ships
     @computer.ships.each do |ship|
       @computer.place_ship(ship, ship.length)
     end
   end
 
-  def render_player_board
-    # @player.collect_ships
-
+  def prompt_player
     puts "(Computer): I have laid out my ships in the grid."
     puts "            You now need to lay out your two ships."
     puts "            The Cruiser is three units long and the"
     puts "            Submarine is two units long."
+    sleep(2)
```

Comment:
Cool implementation of sleep() here! I like that it gives a slight pause to displaying the rendered board, so a person has time to read the computer's message about what they should do next.